libs/util-contracts/src/lib/abiAndAddresses/stakingVerifier.js

```diff
@@ -0,0 +1,398 @@
+export const STAKING_VERIFIER = {
```

Comment:
missing addresses in this object like is often found in the objects in this directory, not a blocker, but may be good to add them for consistency

Reply:
yep you're right, I was thinking they will be created by users, but it's gonna happen on our end. will add them in next prs